src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -25,20 +25,30 @@ public static void Bind(this IConfiguration configuration, object instance)
             }
         }
 
-        public static T Get<T>(this IConfiguration configuration)
+        public static T Get<T>(this IConfiguration configuration, T defaultValue)
```

Comment:
I view it as, I'm Getting a Value from Config.  TValue is the type of the value.  But maybe you have a deeper insight than I do since you are more familiar with Config